<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LyricsProvidersDialog</class>
 <widget class="QDialog" name="LyricsProvidersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lyrics providers</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="providersList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="controlsWidget">
     <layout class="QVBoxLayout" name="controlsLayout">
      <item>
       <widget class="QPushButton" name="upButton"/>
      </item>
      <item>
       <widget class="QPushButton" name="checkButton">
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downButton"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
